source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -188,11 +188,11 @@ def _call_geovelo(self, url, method=requests.post, data=None):
     @staticmethod
     def get_geovelo_tag(geovelo_response):
         if geovelo_response['title'] == 'RECOMMENDED':
-            return 'rapid'
+            return 'balanced'
         elif geovelo_response['title'] == 'FASTER':
```

Comment:
I had mapped "rapid" with this one : rapid is a well-known tag in Navitia, not shorter ;)

Reply:
Yep you'r right, but our front team use the "shortest" tag to highlight the "itinéraire rapide"